src/components/FreelancerCard.astro

```diff
@@ -78,21 +88,34 @@ const freelancerImage: ImageMetadata | undefined = imageFiles[imagePath];
         {`${freelancer.firstName} ${freelancer.lastName}`}
       </h2>
     </a>    <p class="text-pink-500 dark:text-pink-400 mb-2">{freelancer.headline}</p>
-    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2">
-      📍 {freelancer.location} {getFlagEmoji(freelancer.countryCode)}
+    <p class="text-sm text-neutral-500 dark:text-neutral-500 mb-2 flex items-center gap-2">
+      📍 {freelancer.location} 
+      {flagImage && (
```

Comment:
we have so much logic in this FreelancerCard.astro file reg. the the flags -> why not make it an own flag-image component then? :)